docs/user-guide/ingress-path-matching.md

```diff
@@ -3,8 +3,7 @@
 ## Regular Expression Support
 
 !!! important
-    Regular expressions is not supported in the `spec.rules.host` field. The wildcard character '*' must appear by itself as the first DNS label and matches only a single label. You cannot have a wildcard label by itself (e.g. Host ==
-"*").
+    Regular expressions is not supported in the `spec.rules.host` field. The wildcard character '*' must appear by itself as the first DNS label and matches only a single label. You cannot have a wildcard label by itself (e.g. Host == "*").
```

Comment:
escape the asterisk with a backslash `"\"`, otherwise asterisk character will not render in markdown, as I mentioned in previous comment and you can see in the preview page